README.md

```diff
@@ -8,6 +8,8 @@
 
 ## Intro
 
+In the [JavaScript Objects](https://learn.co/lessons/javascript-objects) lesson, we learned that an `Object` is the basic associative data structure in JavaScript. We can use these objects to associate data *values* with unique *keys*, giving us a human-readable representation of a logical collection of data.
```

Comment:
I probably should have mentioned this earlier, but one initiative that I've been trying to head up is modularizing our curriculum a bit more. Could we introduce this content without referencing the previous lesson? Totally okay to repeat ourselves by way of review, just as long as this lesson could stand on its own in some other curriculum somewhere.